test/integration/next/default-pages-dir/test/next-build.test.ts

```diff
@@ -118,9 +108,17 @@ test("next build works", async () => {
   const bunCliOutput = (await new Response(bunBuild.stdout).text())
     // remove timestamps from output
     .replace(/\(\d+(?:\.\d+)? m?s\)/gi, "");
+  // normalize displayed bytes (round down to 0)
+  // .replace(/\d(?:\.\d+)?(?= k?B)/g, "0")
+  // normalize multiple spaces to single spaces (must perform last)
+  // .replace(/\s{2,}/g, " ");
   const nodeCliOutput = (await new Response(nodeBuild.stdout).text())
     // remove timestamps from output
     .replace(/\(\d+(?:\.\d+)? m?s\)/gi, "");
+  // normalize displayed bytes (round down to 0)
+  // .replace(/\d(?:\.\d+)?(?= k?B)/g, "0")
+  // normalize multiple spaces to single spaces (must perform last)
+  // .replace(/\s{2,}/g, " ");
 
```

Comment:
☝️ Prep for fixing, side-stepping this test fail (unrelated to path) but reoccuring fail.